<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UTN WAV Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/icons/icon_utn.png</normaloff>:/icons/icon_utn.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="iconos.qrc">:/icons/fondo_info_cancion.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>301</width>
      <height>8</height>
     </rect>
    </property>
    <property name="maximum">
     <number>35</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_author">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_curr_time">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>101</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1:04</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max_time">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>101</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1:04</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>101</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> /</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_author_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Autor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Titulo:</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_valor_volumen">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>30</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_volumen">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>173</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="iconos.qrc">:/icons/icono_volumen.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_prev">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_prev.png</normaloff>:/icons/icono_prev.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_next">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_next.png</normaloff>:/icons/icono_next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_play">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_play.png</normaloff>:/icons/icono_play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_stop.png</normaloff>:/icons/icono_stop.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_repeat_mode">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_repeat.png</normaloff>:/icons/icono_repeat.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_list">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/icono_lista.png</normaloff>:/icons/icono_lista.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_title</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_author</zorder>
   <zorder>label_curr_time</zorder>
   <zorder>label_max_time</zorder>
   <zorder>label_5</zorder>
   <zorder>label_author_2</zorder>
   <zorder>label_title_2</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label_valor_volumen</zorder>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>horizontalSlider_volumen</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>373</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConectar"/>
    <addaction name="separator"/>
    <addaction name="actionMicroSD"/>
    <addaction name="separator"/>
    <addaction name="actionLista"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMicroSD">
   <property name="text">
    <string>Explorar MicroSD</string>
   </property>
  </action>
  <action name="actionLista">
   <property name="text">
    <string>Lista de reproduccion actual</string>
   </property>
  </action>
  <action name="actionCreador">
   <property name="text">
    <string>Creador de listas de reproduccion</string>
   </property>
  </action>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
